src/frontend/src/btc/components/send/BtcSendAmount.svelte

```diff
@@ -1,10 +1,32 @@
 <script lang="ts">
+	import { isNullish } from '@dfinity/utils';
+	import { getContext } from 'svelte';
 	import type { BtcAmountAssertionError } from '$btc/types/btc-send';
+	import { SEND_CONTEXT_KEY, type SendContext } from '$icp-eth/stores/send.store';
 	import SendInputAmount from '$lib/components/send/SendInputAmount.svelte';
+	import { ZERO } from '$lib/constants/app.constants';
 	import { tokenDecimals } from '$lib/derived/token.derived';
+	import { getMaxTransactionAmount } from '$lib/utils/token.utils';
 
 	export let amount: number | undefined = undefined;
 	export let amountError: BtcAmountAssertionError | undefined;
+
+	const { sendToken, sendBalance } = getContext<SendContext>(SEND_CONTEXT_KEY);
+
+	$: calculateMax = (): number | undefined =>
+		isNullish($sendToken)
+			? undefined
+			: // TODO: Add fee to the calculation
+				getMaxTransactionAmount({
+					balance: $sendBalance ?? ZERO,
```

Comment:
i think you don't need a fallback here (it is included in the function)

I see that it is used with fallback in `SendForm`, but it is not needed there either